charts/jd-public/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: jd-public
```

Comment:
a values.yaml file should be created with https://github.com/hmcts/shared-services-flux/blob/master/k8s/stg/releases/juror-digital/jd-public.yaml#L44-L48

Reply:
added values file with the entries